tests/kuttl/tests/designate_tls/02-redis-tls-certs.yaml

```diff
@@ -0,0 +1,45 @@
+# hardcode the secret generated by the certificate CR below,
```

Comment:
should we delete those comments?

Reply:
I copy pasted it from the redis repo, but perhaps you're right